front-end/src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,22 @@
+import './Banner.css';  
+import { bannerData } from "../../mocks/data";
+
+const banners = bannerData.banners;
+
+export const Banner = () => {  
+
+    return (  
+        <div className="banner-section">  
+            <div className="banner-list">  
+                <div className="banner-list-previous"></div>
+                {  
+                    banners.map((banner, index) =>   
+                        <img key={index} id={`banner-${index}`} className={`banner`} src={require(`../../mocks${banner.image}`)} alt={banner.title} />  
```

Comment:
should not use require here, you can create function getSrcImg(path) and return value